src/main/java/org/ihiw/management/web/rest/UploadResource.java

```diff
@@ -131,6 +131,52 @@ public UploadResource(UploadRepository uploadRepository, FileRepository fileRepo
         }
         return ResponseEntity.badRequest().headers(HeaderUtil.createEntityUpdateAlert(applicationName, true, ENTITY_NAME, upload.getId().toString())).build();
     }
+    
+    
+    /**
+     * {@code PUT  /uploads} : Set validation status on an existing upload.
+     *
+     * @param upload the upload to update.
+     * @return the {@link ResponseEntity} with status {@code 200 (OK)} and with body the updated upload,
+     * or with status {@code 400 (Bad Request)} if the upload is not valid,
+     * or with status {@code 500 (Internal Server Error)} if the upload couldn't be updated.
+     * @throws URISyntaxException if the Location URI syntax is incorrect.
+     */
+    @PutMapping("/uploads/{id}")
```

Comment:
This should only be allowed for some sort of system user. Probably we want to create a new user (manually) with a new type of role "SYSTEM" (this has to go into AuthoritiesConstants.java). Following annotation should then do the job:

`@PreAuthorize("hasRole('" + AuthoritiesConstants.SYSTEM + "')")`

Reply:
Done, the role is called VALIDATION